Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -754,6 +773,16 @@ extension PortfolioStore: PreferencesObserver {
   }
 }
 
+extension PortfolioStore: WalletUserAssetDataObserver {
```

Comment:
Does this need setup similar to observers in `WalletServiceObservers.swift` to not call `update()` when iOS 17.0..<17.2 leaks observed objects in SwiftUI sheets/modals (https://github.com/brave/brave-ios/issues/8160)?

Reply:
tested with no leaks! :)